index.js

```diff
@@ -2,15 +2,35 @@ const express = require('express');
 const app = express();
 const http = require('http').Server(app);
 const io = require('socket.io')(http);
-
 const path = require('path');
+const bodyParser = require('body-parser');
+const Poll = require('./lib/poll');
+
+app.set('view engine', 'ejs');
 
 app.use(express.static('public'));
+app.use(bodyParser.urlencoded({ extended: true }));
 
 app.get('/', function (req, res){
   res.sendFile(path.join(__dirname, '/public/index.html'));
 });
 
+app.get('/new', function (req, res){
+  res.sendFile(path.join(__dirname, '/public/new.html'));
+});
+
+app.post('/create', function (req, res){
+  var poll = new Poll();
+  poll.question = req.body.question;
+  poll.addChoice(req.body.choice1);
+  poll.addChoice(req.body.choice2);
+  poll.addChoice(req.body.choice3);
```

Comment:
This approach isn't really going to be able to handle a variable number of choices. You can structure your choices into an array doing something like [this](https://github.com/turingschool-examples/expressly/blob/master/index.js#L23-L25).